CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+
+cmake_minimum_required (VERSION 2.6)
+project (Bloaty)
+
+set(RE2_BUILD_TESTING OFF CACHE BOOL "enable testing for RE2" FORCE)
+add_subdirectory(third_party/re2)
+include_directories(third_party/re2)
+
+include_directories(.)
```

Comment:
include_directories(src)?

Reply:
Doesn't seem to be required? It is compiling without this.

Reply:
Hmm. I saw that in the source code it's doing 'include "bloaty.h"' so I assume you need to put src in the include path. Is it there by default?